_includes/footer.html

```diff
@@ -1,18 +1,20 @@
-<!-- Footer -->
-<div class="wrapper">
-    <footer class="footer">
-
-        <p class="footer__text">&copy; 2016 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd.</p>
-
-        <ul class="footer__links">
-            <li class="footer__list-item"><a href="http://www.ubuntu.com//legal">Legal information</a><span class="footer__dot"> &#183;</span></li>
-            <li class="footer__list-item"><a href="https://github.com/ubuntudesign/cloud-init/issues/new">Report a bug on this site</a></li>
+<footer class="p-footer">
+  <div class="row">
+    <div class="col-12">
+      &copy; 2017 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd.
+      <nav class="p-footer__nav">
+        <ul class="p-footer__links">
+          <li class="p-footer__item">
+            <a class="p-footer__link" href="http://www.ubuntu.com//legal">Legal information</a>
+          </li>
+          <li class="p-footer__item">
+            <a class="p-footer__link" href="https://github.com/ubuntudesign/cloud-init/issues/new">Report a bug on this site</a>
```

Comment:
Do we need to add ```external``` to these links as well as ```aria-label``` to say its going to an external site?